### DISCUSSION TITLE
Error: extracted_data is not defined

### DISCUSSION THREAD

**Discussion by alandaitch** [omkarcloud/google-maps-scraper#66] [General]
I'm getting this every time I try to run the script new, is there a way to troubleshoot? Maybe my IP got banned?

selenium.common.exceptions.JavascriptException: Message: javascript error: Cannot read properties of null (reading 'querySelector')
  (Session info: chrome=119.0.6045.159)
Stacktrace:
0   chromedriver-119                    0x0000000100cba004 chromedriver-119 + 4169732
1   chromedriver-119                    0x0000000100cb1ff8 chromedriver-119 + 4136952
2   chromedriver-119                    0x0000000100907500 chromedriver-119 + 292096
3   chromedriver-119                    0x000000010090c808 chromedriver-119 + 313352
4   chromedriver-119                    0x000000010090eaf4 chromedriver-119 + 322292
5   chromedriver-119                    0x0000000100988138 chromedriver-119 + 819512
6   chromedriver-119                    0x0000000100987818 chromedriver-119 + 817176
7   chromedriver-119                    0x00000001009405e8 chromedriver-119 + 525800
8   chromedriver-119                    0x00000001009414b8 chromedriver-119 + 529592
9   chromedriver-119                    0x0000000100c80334 chromedriver-119 + 3932980
10  chromedriver-119                    0x0000000100c84970 chromedriver-119 + 3950960
11  chromedriver-119                    0x0000000100c68774 chromedriver-119 + 3835764
12  chromedriver-119                    0x0000000100c85478 chromedriver-119 + 3953784
13  chromedriver-119                    0x0000000100c5aab4 chromedriver-119 + 3779252
14  chromedriver-119                    0x0000000100ca1914 chromedriver-119 + 4069652
15  chromedriver-119                    0x0000000100ca1a90 chromedriver-119 + 4070032
16  chromedriver-119                    0x0000000100cb1c70 chromedriver-119 + 4136048
17  libsystem_pthread.dylib             0x000000018be89034 _pthread_start + 136
18  libsystem_pthread.dylib             0x000000018be83e3c thread_start + 8

Traceback (most recent call last):
  File "/Users/alandaitch/web_scripts/Tests/scrapers/google-maps-scraper-master/main.py", line 30, in <module>
    Gmaps.places(queries, max=5,scrape_reviews=True,lang=Gmaps.Lang.Spanish)
  File "/Users/alandaitch/web_scripts/Tests/scrapers/google-maps-scraper-master/src/gmaps.py", line 306, in places
    result_item = process_result(min_reviews, max_reviews, category_in, has_website, has_phone, min_rating, max_rating, sort, key, scrape_reviews, reviews_max, reviews_sort, fields, lang, should_scrape_socials, places_obj)
  File "/Users/alandaitch/web_scripts/Tests/scrapers/google-maps-scraper-master/src/gmaps.py", line 175, in process_result
    places = places_obj["places"]
TypeError: 'NoneType' object is not subscriptable

**Comment by paulanka2000** [comment]
any solutions for this?

**Comment by Chetan11-dev** [comment]
Released a fix, Please follow these Steps. 

⿡ Clone the Magic 🧙‍♀:
```
git clone https://github.com/omkarcloud/google-maps-scraper
cd google-maps-scraper
```

⿢ Install Dependencies 📦:
```
python -m pip install -r requirements.txt
```

⿣ Let the Rain of Google Map Leads Begin 😎:
```
python main.py 
```

And Run the following command to update packages:

```
python -m pip install botasaurus --upgrade
```

If it occurs again please share `error_log` folder.
